packages/nodejs/src/helpers/cookies.ts

```diff
@@ -25,7 +25,8 @@ export const genCookie = (
     opts.sameSite = 'none';
   }
   if (opts.partitioned === undefined) {
-    opts.partitioned = true;
+    // TODO: Set to true by default after all 3pcd support is complete
```

Comment:
It might be a more effective solution to remove the entire 'if' clause if that results in the expected behavior.